frontend/src/scenes/notebooks/Nodes/NotebookNodeQuery.tsx

```diff
@@ -46,56 +39,38 @@ const Component = (props: NodeViewProps): JSX.Element => {
         return 'Query'
     }, [query])
 
+    useEffect(() => {
+        setTitle(title)
+        // TODO: Set title on parent props
+    }, [title])
+
     const modifiedQuery = useMemo(() => {
         const modifiedQuery = { ...query }
 
         if (NodeKind.DataTableNode === modifiedQuery.kind) {
             // We don't want to show the insights button for now
             modifiedQuery.showOpenEditorButton = false
-            modifiedQuery.full = editing
+            modifiedQuery.full = expanded
         }
         return modifiedQuery
-    }, [query, editing])
+    }, [query, expanded])
 
     return (
-        <NodeWrapper nodeType={NotebookNodeType.Query} title={title} heightEstimate={DEFAULT_HEIGHT} {...props}>
-            <BindLogic logic={insightLogic} props={insightProps}>
-                <Query query={modifiedQuery} setQuery={(t) => setQuery(t as any)} />
-            </BindLogic>
-        </NodeWrapper>
+        <BindLogic logic={insightLogic} props={insightProps}>
+            <Query query={modifiedQuery} setQuery={(t) => setQuery(t as any)} />
+        </BindLogic>
     )
 }
 
-export const NotebookNodeQuery = Node.create({
-    name: NotebookNodeType.Query,
-    group: 'block',
-    atom: true,
-    draggable: true,
-
-    addAttributes() {
-        return {
-            height: {
-                default: DEFAULT_HEIGHT,
-            },
-            query: {
-                default: DEFAULT_QUERY,
-            },
-        }
-    },
-
-    parseHTML() {
-        return [
-            {
-                tag: NotebookNodeType.Query,
-            },
-        ]
-    },
-
-    renderHTML({ HTMLAttributes }) {
-        return [NotebookNodeType.Query, mergeAttributes(HTMLAttributes)]
-    },
-
-    addNodeView() {
-        return ReactNodeViewRenderer(Component)
+export const NotebookNodeQuery = createPostHogWidgetNode({
+    nodeType: NotebookNodeType.Query,
+    title: 'Query', // TODO: allow this to be updated from the component
```

Comment:
Yeah thats was exactly my thinking, although it could still react to the attrs changing 🤔 

Long term I think we should move this into the logic, a little like how "breadcrumbs" works for scenes